test/helpers/property_helpers.ex

```diff
@@ -0,0 +1,30 @@
+defmodule PropertyHelpers do
+  @moduledoc """
+  Custom data generators to be used in property tests
+  """
+
+  def date_time_generator(:struct) do
+    date_time_generator(:tupple)
+    |> StreamData.map(&Timex.to_datetime/1)
+  end
+
+  def date_time_generator(:tupple) do
```

Comment:
I may just be missing something, but why :tupple and not :tuple?

Reply:
@ckhrysze typo :( - thanks for the feedback 👍